ilastik/utility/exportFile.py

```diff
@@ -249,6 +249,9 @@ def create_slicing(axistags, dimensions, margin, feature_table):
         yields also the actual object id
     """
     assert margin >= 0, "Margin muss be greater than or equal to 0"
+    if len(feature_table) == 0:
+        return None
```

Comment:
This function's docstring says that it returns an iterator, but this line returns a `None`. Seems like it should return an empty iterator if that makes sense, or throw an exception if it doesn't.

Reply:
Heya, thanks a lot for picking this up! I've addressed it so you could have another look :)